test/cached-build.test.js

```diff
@@ -52,7 +53,7 @@ test('it should correctly generate all required pino files when the cache is ena
   runBuild(distFolder, (err) => {
     t.error(err)
 
-    runBuild(distFolder, (err, stats) => {
+    runBuild(distFolder, async (err, stats) => {
```

Comment:
__never__ mix callbacks and `async`. Refactor this to just use promises if you think that's necessary.

Reply:
moved to promises

Reply:
I've removed the async and changed the `await` parts to `.then`

Is it ok now?